Generate config files + run download in loop

crangelsmith opened the pull request:
Fixes #31

andrewphilipsmith reviewed (changes requested):
Generally looks good to me.

There is one change that I think should be made (assuming I've understood it correctly). In `generate_config_file.py`, I think the bounds file represents the GEE image downloads, not the chips for inference - Is that right? If so, then the variable name `chips_gdf` is misleading. I've changed this to `bounds_gdf`. I've also tweaked the CLI parameter description to remove the word "chips".

I've added a small commit with these changes. Feel free to revert my change if I've got it wrong.

crangelsmith commented:
> Generally looks good to me.
> 
> There is one change that I think should be made (assuming I've understood it correctly). In `generate_config_file.py`, I think the bounds file represents the GEE image downloads, not the chips for inference - Is that right? If so, then the variable name `chips_gdf` is misleading. I've changed this to `bounds_gdf`. I've also tweaked the CLI parameter description to remove the word "chips".
> 
> I've added a small commit with these changes. Feel free to revert my change if I've got it wrong.

Thank you @andrewphilipsmith, this makes sense! I'll fix the broken test and merge it.
